### DISCUSSION TITLE
The 0 point of the countdown should be pointed to the moment of the flip lottery

### DISCUSSION THREAD

**Discussion by ltraveler** [idena-network/idena-web#137] [Ideas]
I think the official validation time has to be pointed to the time when flip lottery starts. Right now we have an unspoken rule to be logged in at least 5 minutes before the validation.
Many newbies do not know that and they are getting to the system after that unspoken rule because in theory, they are on time if they logged in 3 minutes before for example but in fact, they gonna get the late submission error.
During the time of the flip lottery, the system could show a validation ready loading bar with some validation tour guide, so people can use that 5 minutes to read something useful about idena.
Maybe part of the iDNA from the community wallet could spend on making a comic strip with Idena as the main protagonist who is showing how to pass the validation or telling about the latest news and so on.

**Comment by hlolve** [comment]
IDs that will make validation on web app with prepared flips to be allowed to be ready to participate, and not running own node during validation, must be logged in before flip lottery. Others, like candidates, and suspended will not need that.
